integreat_cms/cms/templates/feedback/region_feedback_list.html

```diff
@@ -49,27 +49,38 @@ <h1 class="heading">{% trans 'Feedback' %}</h1>
                                class="bulk-select-item"/>
                     </td>
                     <td class="pr-2 whitespace-nowrap">{{ feedback.category }}</td>
-                    <td class="pr-2 truncate max-w-[75px] 2xl:max-w-[200px]" title="{{ feedback.object_name }}">
+                    <td class="pr-2 truncate max-w-[75px] 2xl:max-w-[200px] overflow-hidden" title="{{ feedback.object_name }}">
```

Comment:
```suggestion
                    <td class="pr-2 truncate max-w-[75px] 2xl:max-w-[200px]" title="{{ feedback.object_name }}">
```
This is not required anymore since it's now handled by the `feedback-entry` class, right?